tests/test_distributed.py

```diff
@@ -135,7 +135,7 @@ def test_no_span_links_from_invalid_trace_id(self):
         assert len(trace) == 0
 
 
-@scenarios.tracing_config_nondefault_2
+@scenarios.default
```

Comment:
Given that the `setup/test_span_links_from_conflicting_contexts` tests was initially checking for W3C -> DD -> B3, and is now being changed to DD -> W3C -> B3, that test should be testing the same behavior as this test. In that case, I feel that this test here can be removed as it is repetitive. WDYT?